base/REPL.jl

```diff
@@ -958,7 +956,7 @@ type StreamREPL <: AbstractREPL
     waserror::Bool
     StreamREPL(stream,pc,ic,ac) = new(stream,pc,ic,ac,false)
 end
-StreamREPL(stream::IO) = StreamREPL(stream, julia_green, Base.input_color(), Base.answer_color())
+StreamREPL(stream::IO) = StreamREPL(stream, Base.text_color[:green], Base.input_color(), Base.answer_color())
```

Comment:
I'm going to leave the prompts bold. Changing the color to `light_green` still might make sense because it decouples the color from its boldness. However, do the light variants work reliably across platforms?